lib/miyuki/notifier.rb

```diff
@@ -19,7 +19,12 @@ def initialize(notifier = nil)
     end
 
     def notify(title, message)
-      @notifier.notify(title, message) if has_notifier?
+      begin
+        @notifier.notify(title, message) if has_notifier?
+      rescue NoMethodError
+        #libnotify is not installed, maybe.
+        #someone may not want to install gnome on its server :V
```

Comment:
Those tests related to notifications are not implemented yet since I need to find a way to perform them. I suggest you to turn off the notifications for now (yes, removing also `begin ... rescue`).